drone/Dockerfile

```
@@ -0,0 +1,4 @@
+FROM drone/cli
+LABEL io.whalebrew.config.environment '["DRONE_SERVER"]'
+LABEL io.whalebrew.config.environment '["DRONE_TOKEN"]'
```

Unless I am mistaking, labels are not extended but overridden, to get both environment variables you need to do this:

```suggestion
LABEL io.whalebrew.config.environment '["DRONE_SERVER", "DRONE_TOKEN"]'
```